docs/generate-time-series/go.rst

```diff
@@ -0,0 +1,374 @@
+endpoint returns a JSON payload, which contains an ``iss_position`` object
+with ``latitude`` and ``longitude`` data.
+
+.. code-block:: js
+
+   {
+        "message":"success",
+        "timestamp":1591703638,
+        "iss_position":{
+            "longitude":"84.9504",
+            "latitude":"41.6582"
+        }
+    }
+
+The longitude and latitude of the International Space Station changes 
+constantly and is what you want to extract from this payload and insert into
+CrateDB. To parse this JSON, you can create a `struct`_ that can be used to 
+`unmarshal`_ the data into. When you unmarshal JSON into a struct, the 
+function matches incoming object keys to the keys in the struct field name
+or its tag. By default, object keys which don't have a corresponding struct 
+field are ignored.
+
+.. code-block:: js
+
+    type issInfo struct {
+        IssPosition struct {
+            Longitude string `json:"longitude"`
+            Latitude  string `json:"latitude"`
+        } `json:"iss_position"`
+    }
+
+Now, create a function that makes an HTTP GET request to the Open Notify API 
+endpoint and returns longitude and latitude as a `geo_point`_ declaration.
+
+The Go standard library comes with great support for HTTP client and server
+implementations in the `net/http`_ package. In this example you’ll use it to 
+issue an HTTP GET request to the API endpoint. 
+
+Then you will read the response body and unmarshal the JSON into your defined 
+struct ``issInfo``. 
+
+Lastly, you will format the return string and return it. 
+
+.. code-block:: js
+
+    func getISSPosition() (string, error) {
+        var i issInfo
+
+        response, err := http.Get("http://api.open-notify.org/iss-now.json")
+        if err != nil {
+            return "", fmt.Errorf("unable to retrieve request: %v", err)
+        }
+        defer response.Body.Close()
+
+        if response.StatusCode/100 != 2 {
+            return "", fmt.Errorf("bad response status: %s", response.Status)
+        }
+
+        responseData, err := ioutil.ReadAll(response.Body)
+        if err != nil {
+            return "", fmt.Errorf("unable to read response body: %v", err)
+        }
+
+        err = json.Unmarshal(responseData, &i)
+        if err != nil {
+            return "", fmt.Errorf("unable to unmarshal response body: %v", err)
+        }
+
+        s := fmt.Sprintf("(%s, %s)", i.IssPosition.Longitude, i.IssPosition.Latitude)
+        return s, nil
+    }
+
+In your main function, call your ``getISSPosition()`` function and print out the result:
+
+.. code-block:: js
+
+    func main() {
+        pos, err := getISSPosition()
+        if err != nil {
+            log.Fatal(err)
+        }
+
+        fmt.Println(pos)
+    }
+
+Save all your changes and run the code in the command line:
+
+.. code-block:: console
+
+    $ go run main.go
+
+The result should contain your geo_point string:
+
+.. code-block:: js
+
+    (104.7298, 5.0335)
```

Comment:
I would add a bit that mentions how you can run this multiple times to get the new position of the ISS each time (like how you do later when inserting)